### PR TITLE
Add SExt and ZExt instructions

### PR DESCRIPTION
**Pull request by mishazharov**
This PR adds SExt and ZExt instructions + a couple of tests

### CODE REVIEW

File: tests/run-pass/sext.ll [hunk 1]
```diff
@@ -0,0 +1,26 @@
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-pc-linux-gnu"
+
+; Here we zero extend i8 to some an i16
```
Comment (Phantomical) [inline tests/run-pass/sext.ll:4]: Typo? `some an`

File: tests/run-pass/zext.ll [hunk 2]
```diff
@@ -0,0 +1,26 @@
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-pc-linux-gnu"
+
+; Here we zero extend i8 to some an i16
```
Comment (Phantomical) [inline tests/run-pass/zext.ll:4]: Same thing here

**Review by Phantomical — APPROVED** [review]
LGTM! Looks like there's a few typos, up to you if you want to fix them

File: tests/run-pass/zext.ll (hunk 2 above)
Comment (mishazharov) [reply tests/run-pass/zext.ll]: Nice catch!

**Pull request merged**